update-sdk-versions.sh

```
@@ -0,0 +1,127 @@
+done
+
+# Regenerate Cargo.lock files
+echo "Regenerating Cargo.lock files..."
+find . -name "Cargo.toml" -type f | while read -r file; do
+  dir=$(dirname "$file")
+  if [ -f "$dir/Cargo.lock" ]; then
+    if (cd "$dir" && cargo build 2>/dev/null); then
+      echo "  ✓ $dir/Cargo.lock"
+    else
+      echo "  ✗ $dir/Cargo.lock (cargo build failed)"
+      CARGO_ERRORS+=("$dir")
```

Shellcheck (0.11.0)

[info] Modification of CARGO_ERRORS is local (to subshell caused by pipeline).

(SC2030)